### PR TITLE
tests: new spread test for interface gpg-keys

### PR DESCRIPTION
**Pull request by sergiocazzolato**

### CODE REVIEW

File: tests/main/interfaces-gpg-keys/task.yaml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+execute: |
+    echo "The interface is not connected by default"
+    snap interfaces | MATCH "$DISCONNECTED_PATTERN"
+
+    echo "When the interface is connected"
+    snap connect test-snapd-gpg-keys:gpg-keys
+
+    echo "Then the snap is able to run gpg"
+    test-snapd-gpg-keys.sh -c "gpg --list-keys"
```
Comment (stolowski) [inline tests/main/interfaces-gpg-keys/task.yaml]: Hmm, did you forget to add `test-snapd-gpg-keys.sh` file?

File: tests/lib/snaps/test-snapd-gpg-keys/meta/snap.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+name: test-snapd-gpg-keys
+summary: Basic snap to access to gpg keys
+description: A basic snap used to access to gpg keys
+version: 1.0
+
+apps:
+  sh:
+    command: ../../../bin/sh
```
Comment (stolowski) [inline tests/lib/snaps/test-snapd-gpg-keys/meta/snap.yaml]: Not `test-snapd-gpg-keys.sh` ?

File: tests/main/interfaces-gpg-keys/task.yaml (hunk 1 above)
Comment (zyga) [reply tests/main/interfaces-gpg-keys/task.yaml]: No, the `.sh` file is just shell itself.

File: tests/lib/snaps/test-snapd-gpg-keys/meta/snap.yaml (hunk 2 above)
Comment (zyga) [reply tests/lib/snaps/test-snapd-gpg-keys/meta/snap.yaml]: This resolves to `/bin/sh`

File: tests/main/interfaces-gpg-keys/task.yaml (hunk 1 above)
Comment (stolowski) [reply tests/main/interfaces-gpg-keys/task.yaml]: Ah, of course! I got confused, thanks!

**Pull request merged**